release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3510,4 +3510,27 @@ $ oc adm release info 4.11.38 --pullspecs
 [id="ocp-4-11-38-upgrading"]
 ==== Updating
 
+To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-11-39"]
+=== RHSA-2023:2014 - {product-title} 4.11.39 bug fix and security update
+
+Issued: 2023-05-02
+
+{product-title} release 4.11.39, which includes security updates, is now available. Bug fixes included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2023:2014[RHSA-2023:2014] advisory. RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:2013[RHBA-2023:2013] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.11.39 --pullspecs
+----
+
+[id="ocp-4-11-39-bug-fixes"]
+==== Bug fixes
+* Previously, the Ingress Operator failed to publish `router-certs` secret on a cluster with a high number of Ingress Controllers because of the secret size limit of 1 mebibyte. Consequently, the Authentication Operator that uses the `router-certs` secret to access the cluster Ingress domain might not have had the latest certificate to use for OAuth purposes. With this update, the Ingress Operator can generate the default certificate and key for each Ingress Controller, so that the Ingress Operator can publish a `router-certs` secret on the cluster. This update ensures that the Authentication Operator can read and use an up-to-date certificate for OAuth authentication. (link:https://issues.redhat.com/browse/OCPBUGS-8000[*OCPBUGS-8000*])
```

Comment:
* There is a specific secret named "router-certs", so it makes sense to write "_the_ `router-certs` secret".
* There is only one Authentication Operator, and it uses the `router-certs` secret, so I think it is appropriate to write "the Authentication Operator, _which_ uses the `router-certs` secret"¸ using the non-restrictive "which" rather than the restrictive "that".
* The Ingress Operator does generate a certificate and key for each Ingress Controller, but that part was working fine before the bug fix, so it seems confusing to me to say, "With this update, the Ingress Operator can generate" it.  The bug fix is that now the Ingress Operator can _publish_ the `router-certs` secret even if there is a high number of Ingress Controllers.  
* The key change is that the Ingress Operator _no longer_ publishes the certificate and key for each Ingress Controller; the Ingress Operator now only publishes the certificate and key that the Authentication Operator needs, namely the certificate and key for the cluster Ingress domain.

```suggestion
* Previously, the Ingress Operator failed to publish the `router-certs` secret on a cluster with a high number of Ingress Controllers because of the secret size limit of 1 mebibyte. Consequently, the Authentication Operator, which uses the `router-certs` secret to access the cluster Ingress domain, might not have had the latest certificate to use for OAuth purposes. With this update, the Ingress Operator publishes the certificate and key only for the Ingress Controller that owns the cluster Ingress domain, so that the secret doesn't exceed the size limit. This update ensures that the Authentication Operator can read and use an up-to-date certificate for OAuth authentication. (link:https://issues.redhat.com/browse/OCPBUGS-8000[*OCPBUGS-8000*])
```

Reply:
Added changes.